deploy/metrics-server-deployment.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: metrics-server
+  namespace: kube-system
```

Comment:
Changing to the other namespace is problematic:
- addon manager only consider pods running in kube-system
- rescheduler the same
- I strongly believe that there are some other parts of the code (maybe in Kubernetes, definitely outside) that relies on the fact that system components run in kube-system

I'm not against but I don't want to handle all that stuff as a part of metrics-server effort.